### ISSUE TITLE
Energy hyperparameter mismatch in paper

### ISSUE THREAD

**Issue by hyun78** [wetliu/energy_ood#36]
Dear Authors, 

I found mismatch in your paper and implementation in this repo. 
hyperparam for energy min/max (m_in, m_out) is swapped.

In paper, 4.1 Training details
- We use the validation set as in Hendrycks et al. [14] to determine the hyperparameters: m_in is chosen from {−3, −5, −7}, and m_out is chosen from {−15, −19, −23, −27} that minimize FPR95.

Implementation
```python
parser.add_argument('--m_in', type=float, default=-25., help='margin for in-distribution; above this value will be penalized')
parser.add_argument('--m_out', type=float, default=-7., help='margin for out-distribution; below this value will be penalized')
```

I think the implementation is correct, but just in case I want to make sure. 
If possible, it would be better to update arxiv paper. 

Thanks
